data/data/ibmcloud/bootstrap/main.tf

```diff
@@ -21,6 +21,8 @@ resource "ibm_is_instance" "bootstrap_node" {
     security_groups = concat(var.control_plane_security_group_id_list, [ibm_is_security_group.bootstrap.id])
   }
 
+  dedicated_host = length(var.control_plane_dedicated_host_id_list) > 0 ? var.control_plane_dedicated_host_id_list[0] : null
```

Comment:
Is this something that we can just simply doc? It's kind of a similar situation of how bootstrap node gets placed in the first control plane subnet, or how bootstrap node will need to be accounted for as far as resource quotas.

Reply:
Sure. I don't know enough about dedicated hosts to make a judgement call on the impact to the user for a typical dedicated-host use case. If you think that it will rarely be an issue, then let's leave it to docs.